app/ethante/fee_checker.go

```diff
@@ -1,159 +0,0 @@
-// Copyright 2021 Evmos Foundation
-// This file is part of Evmos' Ethermint library.
-//
-// The Ethermint library is free software: you can redistribute it and/or modify
-// it under the terms of the GNU Lesser General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// The Ethermint library is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU Lesser General Public License for more details.
-//
-// You should have received a copy of the GNU Lesser General Public License
-// along with the Ethermint library. If not, see https://github.com/evmos/ethermint/blob/main/LICENSE
-package ethante
-
-import (
-	"fmt"
-	"math"
-
-	errorsmod "cosmossdk.io/errors"
-	sdkmath "cosmossdk.io/math"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	errortypes "github.com/cosmos/cosmos-sdk/types/errors"
-	authante "github.com/cosmos/cosmos-sdk/x/auth/ante"
-	ethermint "github.com/evmos/ethermint/types"
-	"github.com/evmos/evmos/v11/x/evm/types"
-)
-
-// NewDynamicFeeChecker returns a `TxFeeChecker` that applies a dynamic fee to
-// Cosmos txs using the EIP-1559 fee market logic.
-// This can be called in both CheckTx and deliverTx modes.
-// a) feeCap = tx.fees / tx.gas
-// b) tipFeeCap = tx.MaxPriorityPrice (default) or MaxInt64
-// - when `ExtensionOptionDynamicFeeTx` is omitted, `tipFeeCap` defaults to `MaxInt64`.
-// - when london hardfork is not enabled, it fallbacks to SDK default behavior (validator min-gas-prices).
-// - Tx priority is set to `effectiveGasPrice / DefaultPriorityReduction`.
-func NewDynamicFeeChecker(k DynamicFeeEVMKeeper) authante.TxFeeChecker {
```

Comment:
we'll need this for #1266

Reply:
Yeah @GAtom22 was looking at that PR on Friday but some problems arrived so decided to remove it not to block the migration. But can add it back if you are ok with it being there unused for the meantime